apps/condo/domains/notification/schema/PushToken.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Generated by `createschema notification.PushToken PushToken`
+ */
+
+const { Text, Relationship, Select } = require('@keystonejs/fields')
+
+const { GQLListSchema } = require('@core/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted } = require('@core/keystone/plugins')
+
+const { SENDER_FIELD, DV_FIELD } = require('@condo/domains/common/schema/fields')
+const access = require('@condo/domains/notification/access/PushToken')
+
+const { DEVICE_SERVICE_TYPES_KEYS } = require('../constants')
+
+
+const PushToken = new GQLListSchema('PushToken', {
+    schemaDoc: 'Push token',
+    fields: {
+        dv: DV_FIELD,
+        sender: SENDER_FIELD,
+
+        user: {
+            schemaDoc: 'to User',
+            type: Relationship,
+            ref: 'User',
+            isRequired: false,
+            kmigratorOptions: { null: true, on_delete: 'models.CASCADE' },
+            knexOptions: { isNotNullable: false }, // token can be crated with anonymous and User could be connected later.
+        },
+
+        deviceId: {
+            schemaDoc: 'Device ID',
```

Comment:
It's unclear, what device and what id. Please, explain in more detail what it is.

Reply:
Same question

Reply:
fixed